src/propagation/jetcoeffs.jl

```diff
@@ -139,26 +139,25 @@ function TaylorIntegration._allocate_jetcoeffs!(::Val{RNp1BP_pN_A_J23E_J2S_ng_ep
     accY = Taylor1(identity(constant_term(zero_q_1)), order)
     accZ = Taylor1(identity(constant_term(zero_q_1)), order)
     local M_ = Array{S}(undef, 3, 3, N)
-    local M_ .= zero_q_1
-    local M_[:, :, ea] .= (t2c_jpl_de430(dsj2k))[:, :] .+ zero_q_1
+    local M_[:, :, ea] = t2c_jpl_de430(dsj2k) .+ zero_q_1
```

Comment:
Why not the `.=`? Performance?

Reply:
I recall that @PerezHz changed this manually some PR's ago. I probably undid this change when updating `jetcoeffs`. Anyhow, I've restored this change.